### ISSUE TITLE
[Bug in flex_space_on_navbar_replace_with_separator.css] Spaces as separators doesn't seem to work in the last version

### ISSUE THREAD

**Issue by unalignedcoder** [Aris-t2/CustomCSSforFx#591]
**1. Does the issue occur without any files inside /chrome/ folder?**
No.

**2. Issue description, userChrome.css/userContent.css 'settings' the issue occurs with and steps to reproduce the issue:**

- Had 4.1.8, decided to upgrade to 4.3.5.
- Replaced the css folders, and edited `userChrome` and `userContent` so that they reflected the same settings I had with version 4.1.8.
- Result: The for `@import "./css/generalui/flex_space_on_navbar_replace_with_separator.css"` setting doesn't work. 
- In both versions this customization is enabled, but in version 4.3.5 doesn't apply.
- **Spaces on the toolbar now remain spaces.**

**4. Does the issue occur with the most recent version of CustomCSSforFx (release/test/dev build)?**
Yes, 4.3.5

**5. Screenshots showing the issue (drag & drop images into this post):**
v. 4.1.8: 
![image](https://user-images.githubusercontent.com/16850566/233774061-d6843511-dc76-411a-8201-8a800ebb4d96.png)
v 4.3.5: 
![image](https://user-images.githubusercontent.com/16850566/233774584-96d7fea7-7351-4ccc-b8d7-de03f02b92d7.png)

**6. System information**
- OS & OS version: Windows 7 x64
- DPI / HiDPI (e.g. 100%, 125%...): 100%
- Firefox version: 1130b6
- Firefox theme (e.g. default, build in light/dark, external theme + link): [Simple Stylish](https://addons.mozilla.org/en-US/firefox/user/7721222/?utm_source=firefox-browser&utm_medium=firefox-browser&utm_content=addons-manager-user-profile-link)
- Is 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets' set to 'true': YES
- Is 'about:config > layout.css.moz-box-flexbox-emulation.enabled' set to 'false' (Fx 108+): YES

(note: there may be further problems, one is custom bookmark folder icons that I had set in `my_userChrome.css`, that worked with 4.1.8, but do not in the new version)

**Comment by Aris-t2** [comment]
Thanks for reporting.

Replace the content of `flex_space_on_navbar_replace_with_separator.css` with this code:

```css
#main-window:not([customizing]) #nav-bar toolbarspring {
  appearance: auto !important;
  -moz-default-appearance: separator !important;
  min-width: 1px !important;
  max-width: 1px !important;
  margin-inline-start: 1px !important;
}
```

**Comment by Aris-t2** [comment]
This was fixed in v4.3.6.
